### PR TITLE
Fastlane-CD #3

### PR DESCRIPTION
**Pull request by bartguminiak**

### CODE REVIEW

File: ELSpace.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -486,13 +490,17 @@
 			baseConfigurationReference = E72147DECCA8EAEE7B4DBA7C /* Pods-ELSpace.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer: Dariusz Rybicki (KB57E8RV46)";
```
Comment (Drwalcore) [inline ELSpace.xcodeproj/project.pbxproj]: Are You sure that we should share this information in **Public Repository**?

File: ELSpace.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -501,13 +509,17 @@
 			baseConfigurationReference = 1E21319772B1B80346FB867E /* Pods-ELSpace.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer: Dariusz Rybicki (KB57E8RV46)";
```
Comment (Drwalcore) [inline ELSpace.xcodeproj/project.pbxproj]: Are You sure that we should share this information in **Public Repository**?

File: fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    # snapshot
+    gym(scheme: "ELSpace") # Build your app - more options available
+    deliver(force: true)
+    # frameit
+  end
+
+  # You can define as many lanes as you want
+
+  after_all do |lane|
+    # This block is called, only if the executed lane was successful
```
Comment (sxcore) [inline fastlane/Fastfile:39]: I think It would be cool to send message to our Slack with successful lane?

**Review by sxcore — CHANGES_REQUESTED** [review]
We will talk about my concerns first thing in the morning on Monday. Besides that, everything looks really good and legit 🥇

File: fastlane/Fastfile (hunk 3 above)
Comment (bartguminiak) [reply fastlane/Fastfile:39]: :+1: I will add it as new feature

**Pull request merged**